plugins/modules/azure_rm_keyvaultsecret_info.py

```diff
@@ -267,7 +280,7 @@ def exec_module(self, **kwargs):
             if self.show_deleted_secret:
                 self.results['secrets'] = self.get_deleted_secret()
             else:
-                if self.version == 'all':
+                if self.list_properties:
```

Comment:
Isn't this introducing an inconsistent behavior between get one kv and list kvs, when this new `list_properties` is specified. Also, this changes the behavior of the previous `self.version == 'all'`.